db/migrate/20200120000001_add_external_wi_id_to_incident.rb

```diff
@@ -0,0 +1,42 @@
+class AddExternalWiIdToIncident < ActiveRecord::Migration[5.1]
+    def up
+  
+      # return if it's a new setup
+      return if !Setting.find_by(name: 'system_init_done')
+  
+      add_column :tickets, :external_ticket_id, :string, limit: 150, null: true
+
+      ObjectManager::Attribute.add(
+        force:       true,
+        object:      'Ticket',
+        name:        'external_ticket_id',
+        display:     'External ticket id',  
+        data_type:   'input',
+        data_option: {
+            default:    '', 
+            maxlength: 150,
+            null:      true,
+            type:      'text'
+        },
+        editable:    false,
+        active:      true,
+        screens:     {
+            create_middle: {},
+            edit: {
+            'ticket.customer' => {
+                shown: true,                           
+            },
+            'ticket.agent' => {
+                shown: true,
+                },
+            },    
+        },
+        to_create:   false,
+        to_migrate:  false,
+        to_delete:   false,
+        position:    103,  
```

Comment:
103 are already in use. Please change to 104

Reply:
updated